Catappult/appcoins-wallet-android
# Google Install Referrer, installation information

**What does this PR do?**
Integrates with Google Install Referrer to identify where the wallet was downloaded from, before google play was opened, by logging:
 - package_name of the game, 
 - sku of the item to buy, 
 - integration_flow (only OSP for now),  
 - source (only google play for now)

**How should this be manually tested?**
Testing this PR is somewhat tricky because this change applies to the first time launching of the wallet, installed from Google Play.
To test this PR follow this steps by this order:
 - uninstall the wallet, 
 - start a osp payment on a game and choose google play in the web page (or click on this example url to open google play directly), 
 - minimize google play, 
 - install the wallet version to test, 
 - reopen the google play (that is already minimized), 
 - click on open to launch the wallet, 
 - the event wallet_first_launch will be sent with package_name, integration_flow, source, sku. Check the logs for this event

**What are the relevant tickets?**
https://aptoide.atlassian.net/browse/APPC-3120

**Code Review Checklist**
